Software/Development/SemesterProject/App/Include/referenceFrameConversion.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * referenceFrameConversion.h
+ *
+ *  Created on: Apr 9, 2019
+ *      Author: Nicolas
+ */
+
+#ifndef APP_INCLUDE_REFERENCEFRAMECONVERSION_H_
+#define APP_INCLUDE_REFERENCEFRAMECONVERSION_H_
+
+
+/*
+ * Temporary declarations To be removed
+ */
+#include "analogAcquisitionManager.h"
+void runClosedLoopControl(void);
+/*
+ * end of temporaty declarations
+ */
+
+
+typedef struct
+{
+    float DComponent;
+    float QComponent;
+}DQObject;
```

Comment:
dComponent, qComponent and dqObject?¿

Reply:
I see you called everything with capital letters, i think it's better with lower case but it's not a big deal so up to you

Reply:
I actually agree with Aitor¿?¿

Reply:
Done